ambry-replication/src/main/java/com.github.ambry.replication/ReplicationManager.java

```diff
@@ -377,6 +377,28 @@ public void start() throws ReplicationException {
   }
 
   /**
+   * Enables/disables {@code types} of replication on the given {@code ids}. The disabling is in-memory and therefore
+   * is not valid across restarts.
+   * @param ids the {@link PartitionId}s to enable/disable it on.
+   * @param origins the list of datacenters from which replication should be enabled/disabled.
```

Comment:
I think that confusion is always going to be there - `srcDCs` can also refer to the "source of the requests". I tried hard to think of names that make sense but I think every name is going to end up being somewhat confusing. So I have tried to make the docs clear.